c7n/resources/glue.py

```diff
@@ -178,6 +178,92 @@ class resource_type(TypeInfo):
     augment = universal_augment
 
 
+class SecurityConfigFilter(ValueFilter):
```

Comment:
this should be updated to use a related resource filter, which will remove most of the code.

Reply:
Do you mean my filter using  _GlueSecurityConfiguration_ (newly added by #5523) in it to be more condense and DRY? 

One side note about _GlueSecurityConfiguration_. Although very useful, I have some use cases (that I think would be fairly common) that it would not cover.  What if you had a glue crawler that you wanted to take action on based on the security configuration. I would have to specify _resource: glue-crawler_ in policy, I'm not aware of way to filter on _glue-security-configuration_ and then take actions on _glue-crawler_.